packages/rocketchat-lib/server/lib/notifyUsersOnMessage.js

```diff
@@ -112,7 +113,9 @@ RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
 	// Update all other subscriptions to alert their owners but witout incrementing
 	// the unread counter, as it is only for mentions and direct messages
 	RocketChat.models.Subscriptions.setAlertForRoomIdExcludingUserId(message.rid, message.u._id);
+	RocketChat.models.Subscriptions.setOpenForRoomIdExcludingUserId(message.rid, message.u._id);
```

Comment:
we now set `alert` and `open` properties in two separate update commands. during tests this proved to be more efficient on MongoDB - because it uses a more efficient index.